From d239bca13f894db7fbf56d14915abae803f28f2d Mon Sep 17 00:00:00 2001
From: Harald Hoyer <harald@redhat.com>
Date: Fri, 5 Aug 2011 13:18:08 +0200
Subject: [PATCH] 99base/selinux-loadpolicy.sh: execute chrooted commands with
 LANG=C

the console is not yet setup correctly to display e.g. japanese

https://bugzilla.redhat.com/show_bug.cgi?id=712254
---
 modules.d/99base/selinux-loadpolicy.sh | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/modules.d/99base/selinux-loadpolicy.sh b/modules.d/99base/selinux-loadpolicy.sh
index 2a19fe1..b82c859 100755
--- a/modules.d/99base/selinux-loadpolicy.sh
+++ b/modules.d/99base/selinux-loadpolicy.sh
@@ -24,11 +24,11 @@ rd_load_policy()
         # load_policy does mount /proc and /selinux in 
         # libselinux,selinux_init_load_policy()
         if [ -x "$NEWROOT/sbin/load_policy" ]; then
-            out=$(chroot "$NEWROOT" /sbin/load_policy -i 2>&1)
+            out=$(LANG=C chroot "$NEWROOT" /sbin/load_policy -i 2>&1)
             ret=$?
 	    info $out
         else
-	    out=$(chroot "$NEWROOT" /usr/sbin/load_policy -i 2>&1)
+	    out=$(LANG=C chroot "$NEWROOT" /usr/sbin/load_policy -i 2>&1)
 	    ret=$?
 	    info $out
         fi
@@ -41,7 +41,7 @@ rd_load_policy()
 	    # If machine requires a relabel, force to permissive mode
 	    [ -e "$NEWROOT"/.autorelabel ] && ( echo 0 > "$NEWROOT"/selinux/enforce )
             mount --bind /dev "$NEWROOT/dev"
-            chroot "$NEWROOT" /sbin/restorecon -R /dev
+            LANG=C chroot "$NEWROOT" /sbin/restorecon -R /dev
 	    return 0
 	fi
 
